docs/development.md

```diff
@@ -64,11 +64,16 @@ GET /api/v1/conversions
         "created": "2022-04-06T06:45:11.83226503Z",
         "finished": "2022-04-06T06:45:11.948393604Z",
         "source": "192.168.1.1/library/nginx:latest",
+        "source_image_size": "67.2 MB",
+        "target_image_size": "69.1 MB",
         "status": "$status",
         "reason": "$reason"
     }
 ]
 ```
+`source_image_size`: string, total size of the source image with specified platforms.
```

Comment:
How about renaming it `source_size`, `target_size`, and using bytes unit by uint type.

Reply:
In my point, we can use `go-humanize` to parse bytes unit to string. It looks more friendly.

Reply:
Renamed.

Reply:
The metrics are typically used by services like prometheus, and it is easier to use them for statistics if they are accurate to the byte and use an integer.